src/pytypes/onefuzztypes/enums.py

```diff
@@ -85,10 +85,8 @@ class TaskFeature(Enum):
 class ContainerPermission(Enum):
     Read = "Read"
     Write = "Write"
-    Create = "Create"
```

Comment:
Were these just removed and subsumed by `Write`? Any docs we can link to?

Reply:
Create doesn't exist as a container permission exposed by the SDK any longer.  

https://docs.microsoft.com/en-us/python/api/azure-storage-blob/azure.storage.blob.containersaspermissions?view=azure-python